enterprise-registry/github-build/index.md

```diff
@@ -81,4 +81,6 @@ Because tagging is [usually done client side](https://docs.docker.com/userguide/
 
 <img src="{{site.url}}/docs/enterprise-registry/github-build/new-tag.png" class="img-center" alt="Create a new tag"/>
 
+Trying to find a specific image for a commit after it is no longer tagged as latest? The build name (visible in the build history of a repository) will match the commit SHA value.
```

Comment:
Does this make sense?

"Trying to match GitHub commits with the resulting container build? The build history of your container will show the output of each build, named by the first 7 digits of your GitHub commit SHA value."